src/js/components/AccountDropdownTrigger.js

```diff
@@ -0,0 +1,43 @@
+import mixin from "reactjs-mixin";
+import PropTypes from "prop-types";
+import React from "react";
+import { StoreMixin } from "mesosphere-shared-reactjs";
+
+class AccountDropdownTrigger extends mixin(StoreMixin) {
```

Comment:
:art: I think @DanielMSchmidt removed the store from his PR, shall we also remove it here? 🤔 (only applicable if he did, not blocking)

Reply:
remove, he did not